UnityProject/Assets/Scripts/Core/Random13.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+
+namespace Core
+{
+	/// <summary>
+	/// An extremely fast, efficient and deterministic class for randomness.
+	/// Best reserved for server logic, while client logic can use DMMath.
+	/// </summary>
+	public class Random13
+	{
+		public Random13()
+		{
+			Instance = this;
+		}
+
+		public static Random13 Instance;
+
+		//https://github.com/id-Software/DOOM/blob/77735c3ff0772609e9c8d29e3ce2ab42ff54d20b/linuxdoom-1.10/m_random.c#LL31C1-L31C1
+		private readonly byte[] randomTable = new byte[]
```

Comment:
Isn't there seed options for this stuff?

Reply:
nope

Reply:
You don't need a seed to change the results of table that will never change

Reply:
That's not how PRNG seeding works. You're still "seeding" the PRNG. It just turns out the seed you picked is a `0`. The exact same can be done with other PRNGs.